src/controllers/group.ts

```diff
@@ -19,8 +19,9 @@ export const show = async (req: Request, res: Response) => {
   const group = await getGroup(+req.params.id)
   if (group) {
     const joined = group.users.some(u => u.id === (req.user as User).id)
+    const owner = group.owner.id === (req.user as User).id
```

Comment:
It would be better if we named it to `isOwner` or something similar. `owner` is a User, not a boolean value.